Удаление лога при обновлении модификаторов

shchs opened the pull request:
Иначе при работе накапливается такой размер лога, что раздел модификаторов, а тем более вкладка лога просто перестает открываться, приходится удалять файл по ftp, наблюдая его размеры, которые легко могут достигать 400Мб

pavlickm commented:
а в каком браузере это работает? пробовал chrome, FF
приходит ссылка с <svg onload=prompt(1234)>
но promt не срабатывает

pavlickm commented:
\<svg onload=prompt(1234)>

halfhope commented:
Извиняюсь, во всех браузерах не работает, только что сам проверил. Как оказалось у меня в FF через дополнение hack bar включена интерпретация JS кода даже для json контента. 
![image](https://cloud.githubusercontent.com/assets/2343320/8925736/bc577258-3523-11e5-8f33-650120b631da.png)
![image](https://cloud.githubusercontent.com/assets/2343320/8925742/c6a26998-3523-11e5-9a0a-3889f9854321.png)

pavlickm commented:
лишним не будет )

shchs committed:
Удаление лога при обновлении модификаторов

Иначе при работе накапливается такой размер лога, что раздел модификаторов, а тем более вкладка лога просто перестает открываться, приходится удалять файл по ftp, наблюдая его размеры, которые легко могут достигать 400Мб
